README.md

```diff
@@ -25,7 +25,7 @@ Power TiDB with your most favorite engines. TiDB supports most of the popular st
 
 ## Status
 
-TiDB is at its early age and under heavy development, some of the features mentioned above have not been fully implemented.
+TiDB is at its early age and under heavy development, all of the features mentioned above are fully implemented.
```

Comment:
Should we update `Multiple storage engine support` section  to add some content about hbase above?